frontend/src/app.css

```
@@ -711,52 +737,168 @@ body {
+.folder-upload-progress {
+  height: 0.8rem;
+  background: rgba(255, 255, 255, 0.08);
+}
+
+.folder-upload-message {
+  color: #f8f9fa;
+}
+
+.folder-upload-detail {
+  line-height: 1.35;
+  word-break: break-word;
```

Stylelint (17.9.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)